src/ResourceManager/ResourceManager/Commands.ResourceManager/Cmdlets/Implementation/TestAzureResourceGroupCmdlet.cs

```diff
@@ -57,6 +57,7 @@ protected override void OnProcessRecord()
         /// </summary>
         private void RunCmdlet()
         {
+            this.WriteWarning("This cmdlet is being deprecated. ");
```

Comment:
IMO we should change this to follow the standard PS cmdlets deprecation message as follows - "The Test-AzureResourceGroup cmdlet is deprecated and will be removed in a future release.”